services/py-api/src/server/handlers/participants_handlers.py

```diff
@@ -1,11 +1,15 @@
+from typing import Optional
 from result import is_err
+from src.server.schemas.jwt_schemas.jwt_user_data_schema import JwtUserData
```

Comment:
Change JwtUserData to have: 
the following fields: 
```Python
# All the types within this should be required
class JwtUserData(BaseTypedDict):
    sub: str
    is_admin: bool
    team_name: str
    is_invite: bool
    exp: datetime
```

Reply:
Fixed in commit a9a6b60a1c533cba63bd89fc5bee4362218c00f7